src/main/java/com/laba/solvd/task2/components/CatalogComponent.java

```diff
@@ -0,0 +1,44 @@
+package com.laba.solvd.task2.components;
+
+import com.laba.solvd.task2.pages.ProductPage;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.Optional;
+
+public class CatalogComponent extends AbstractUIObject {
+    @FindBy(xpath = ".//*[contains(@class,'menu-categories__link')]")
+    private List<ExtendedWebElement> categories;
+    @FindBy(xpath = ".//li[contains(@class,'menu-categories__item')]" +
+            "//*[@class='menu__main-cats']//ul[contains(@class,'menu__main-brands')]" +
+            "//*[contains(@class,'brand-link')]")
```

Comment:
very complicated locator. try to  use just     .//li[contains(@class,'main-brand')]/a